app/app/users/page.tsx

```diff
@@ -48,21 +68,29 @@ export default function Users() {
                 <Button
                   className="me-2"
                   onClick={() => setShowNewUserModal(true)}
+                  disabled={isValidating}
```

Comment:
disable the buttons while we are showing the `isValidating` spinner (different from the `isLoading` spinner which is for loading all the users in the table)